lms/djangoapps/teams/tests/test_views.py

```diff
@@ -509,7 +516,7 @@ def test_expand_public_user(self):
         ('queryable', ['Search']),
         ('Tonga', ['Search']),
         ('Island', ['Search']),
-        ('search queryable', []),
+        ('not-a-query', []),
         ('team', ['Another Team', 'Public Profile Team']),
```

Comment:
Oh yeah, good point. I hadn't even considered that. Since this PR is already based off the instructor tools PR, I'll go ahead and include the relevant types of `learner_removed` events:
 * Team is deleted, all memberships removed
 * Instructor uses tools to remove a membership

That doesn't touch "a learner leaves the team", but I could add that here too without too much trouble if you think that's the way to go.